src/Event/FieldEncrypting.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\EventSourcing\Event;
+
+interface FieldEncrypting
+{
+    public static function getEncryptingFields(): EncryptingFields;
```

Comment:
Why make this a static function?

Reply:
It must be usable on a non initialised object eg `$eventName:: getEncryptingFields()`

Reply:
Makes sense